src/components/viewers/MultiFlatmap.vue

```diff
@@ -140,6 +139,15 @@ export default {
         }
       }
     },
+    displayTooltip: function(info) {
+      let name = info.name;
```

Comment:
Is info always defined? Would throw an error if not

Reply:
True.